recipe/meta.yaml

```diff
@@ -80,35 +80,35 @@ source:
       - patches/0033-gh24324.patch
 
   # TODO :: Depend on our own packages for these:
-  - url: https://github.com/python/cpython-source-deps/archive/xz-5.2.2.zip          # [win]
-    folder: externals/xz-5.2.2                                                       # [win]
-    sha256: 02b6d6f1e0369940e7e1f0d324992e069d09f5f3d32d86e2d22cd8b82ac3fb3f         # [win]
+  - url: https://github.com/python/cpython-source-deps/archive/xz-5.2.2.zip  # [win]
+    folder: externals/xz-5.2.2  # [win]
+    sha256: 02b6d6f1e0369940e7e1f0d324992e069d09f5f3d32d86e2d22cd8b82ac3fb3f  # [win]
```

Comment:
Do we know why it is changing the whitespace here?

Reply:
this is the bot. it normalizes yaml